packages/atomic/src/components/product-listing/atomic-product-listing-result-template/atomic-product-listing-result-template.tsx

```diff
@@ -0,0 +1,69 @@
+import {ResultTemplate, ResultTemplateCondition} from '@coveo/headless';
```

Comment:
Similar problem as described above with mixing imports from `@coveo/headless` which contains all the code for "classic" search interface (and so potentially polluting the `product-listing` use case with un-necessary code).

Take a look at what was done for `atomic-insight-result-template`